src/components/VerifyUser/VerifyUser.scss

```diff
@@ -0,0 +1,50 @@
+$error-color: #ff0000;
+$bg-color: #ffffff;
+$text-color: #ccc;
+$form-color: #f3994a;
+$success-color: #008000;
+
+.containers {
+  align-content: center;
+  display: flex;
+  flex-direction: column;
+  height: calc(100vh);
+  justify-content: center;
+  padding: 0;
+  width: 100%;
+  z-index: 1000;
+
+  .verified {
+    padding: 3rem;
+    height: 140px;
+  }
+
+  @media screen and (min-width: 1200px) {
+    max-width: 2140px;
+    padding: 0;
+  }
+
+  .bg-image {
+    background: url("../../assets/images/background.jpg") center center no-repeat;
+    background-position: center;
+    background-repeat: no-repeat;
```

Comment:
The shorthand`background` property has already taken care of `background-position` and `background-repeat`.

This is the shorthand property 👇 
```css
background: url("../../assets/images/background.jpg") center center no-repeat;
```
So get rid of 
```css
background-position: center;
background-repeat: no-repeat;
```

Reply:
okay. would do that